kroxylicious-filters/kroxylicious-simple-transform/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.kroxylicious</groupId>
+        <artifactId>kroxylicious-parent</artifactId>
+        <version>0.4.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>kroxylicious-simple-transform</artifactId>
```

Comment:
I hesitate about the name. its accurate I agree but not really useful for production usage so maybe we should call them example or dummy? Or am I over thinking it?

Reply:
Agreed of course this is not intended for production use. I can add a statement to the javadoc just to make that clear.

Not keen on the word 'dummy'.  It implies to me that the filter doesn't actually do anything i.e. a null transform.  I prefer 'simple'.